spec/setup_acceptance_node.pp

```diff
@@ -1,12 +1,5 @@
-$major = $facts['os']['release']['major']
-
-# Defaults to staging, for release, use
-# $baseurl = "https://yum.theforeman.org/releases/nightly/el${major}/x86_64/"
-$baseurl = "http://koji.katello.org/releases/yum/foreman-nightly/RHEL/${major}/x86_64/"
```

Comment:
there is no good way to use koji with `foreman::repo` -- do we really need it?

Reply:
Personally I think it's not that important and I'd be happy to only use releases. Previously we needed unreleased things because Katello was broken for too long but I think we should work harder to keep it green instead.